server/views/pages/bookings/cancel/cancel.njk

```diff
@@ -25,15 +25,17 @@
 
       <h2 class="govuk-heading-m">Prisoner</h2>
 
-      <p class="govuk-!-margin-bottom-1" data-test="prisoner-name">{{ prisoner.firstName | capitalize }} {{ prisoner.lastName | capitalize }}</p>
+      <p data-test="prisoner-name">{{ prisoner.firstName | capitalize }} {{ prisoner.lastName | capitalize }}</p>
 
       <h2 class="govuk-heading-m">Visitors</h2>
 
       {% for visitor in visit.visitors %}
-        <p data-test="visitor-name-{{ loop.index }}">{{ visitor.firstName }} {{ visitor.lastName }}</p>
+        <p class="{{ "govuk-!-margin-bottom-1" if not loop.last }}" data-test="visitor-name-{{ loop.index }}">
```

Comment:
Don't apply the  bottom margin style override on the last (or only) visitor listed - otherwise it's too close to the next heading.